Guide.docc/MigrationGuide.md

```diff
@@ -14,27 +14,27 @@
 
 Swift's concurrency system, introduced in [Swift 5.5](https://www.swift.org/blog/swift-5.5-released/),
 makes asynchronous and parallel code easier to write and understand.
-With the Swift 6 language mode, the compiler can now also
-guarantee that concurrent programs are data-race-free.
-When this mode is enabled, the compiler safety checks that were
-optional become required.
+With the Swift 6 language mode, the compiler can now 
+guarantee that concurrent programs are free of data races.
+With this mode enabled, compiler safety checks that were
+previously optional become required.
 
-Adopting the Swift 6 language mode is entirely under your control,
+Adopting the Swift 6 language mode is entirely under your control
 on a per-target basis.
-Targets that build with previous modes, as well as code that uses other
+Targets that build with previous modes, as well as code in other
 languages exposed to Swift, can all interoperate with
 modules that have been migrated to Swift 6.
 
-It is possible you have been incrementally adopting all concurrency features
+It is possible you have been incrementally adopting concurrency features
```

Comment:
I think the intent of the 'all' was to convey 'whether you've been all-in on adopting concurrency, or you haven't done anything at all.'

I think the 'all' is implied in 'incrementally adopting concurrency features as they were introduced' and reads better.

Plus without the 'all' I think it reads as more inclusive for folks who have adopted some concurrency features already.